pages/data-providers/[data-provider-id]/deposit-dates.css

```diff
@@ -41,23 +34,44 @@
   margin-bottom: 0;
   overflow-x: auto;
 
-  & > table {
-    min-width: 40rem;
```

Comment:
I would keep `min-height` here. Otherwise, it causes a horizontal overflow on small screens.

Reply:
You mean min-width?

Reply:
yes sorry

Reply:
I kept it. Thank you!